### PR TITLE
feat: Project collaboration, joined projects, efficiency measure/bug i18n

### PR DESCRIPTION
**Pull request by shuofan**
#### What this PR does / why we need it:
Project collaboration, joined projects, efficiency measure/bug i18n

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @Effet 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |   i18n           |
| 🇨🇳 中文    |      国际化补充        |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: modules/dop/component-protocol/components/auto-test-space-list/components/spaceList/render.go [hunk 1]
```diff
@@ -26,10 +26,10 @@ import (
 	"github.com/erda-project/erda/bundle"
 	"github.com/erda-project/erda/modules/dop/component-protocol/components/auto-test-space-list/common"
 	"github.com/erda-project/erda/modules/dop/component-protocol/components/auto-test-space-list/i18n"
+	text "github.com/erda-project/erda/modules/dop/component-protocol/components/common"
```
Comment (Effet) [inline modules/dop/component-protocol/components/auto-test-space-list/components/spaceList/render.go:29]: why text

File: modules/dop/component-protocol/components/requirement-task-overview/stackChartBlock/stackChartFilter/render.go [hunk 2]
```diff
@@ -59,7 +60,7 @@ func (f *Filter) Render(ctx context.Context, c *cptype.Component, scenario cptyp
 				},
 				Key:       "type",
 				Label:     "类型",
```
Comment (Effet) [inline modules/dop/component-protocol/components/requirement-task-overview/stackChartBlock/stackChartFilter/render.go:62]: i18n

**Review by Effet — CHANGES_REQUESTED** [review]

File: modules/dop/component-protocol/components/requirement-task-overview/container/simpleChart/render.go [hunk 3]
```diff
@@ -44,7 +45,7 @@ func (s *SimpleChart) Render(ctx context.Context, c *cptype.Component, scenario
 	s.Type = "SimpleChart"
 	s.Data = Data{
 		Main:        strconv.Itoa(len(s.Issues)),
-		Sub:         "总数",
+		Sub:         cputil.I18n(ctx, "total"),
 		CompareText: "较昨日",
```
[the remark this answers is not in the document]
Comment (shuofan) [reply modules/dop/component-protocol/components/requirement-task-overview/container/simpleChart/render.go:49]: requirement task overview i18n will be in another pr

File: modules/dop/component-protocol/components/auto-test-space-list/components/spaceList/render.go (hunk 1 above)
Comment (shuofan) [reply modules/dop/component-protocol/components/auto-test-space-list/components/spaceList/render.go:29]: alias of text content, another common package has already been used

File: modules/dop/component-protocol/components/requirement-task-overview/stackChartBlock/stackChartFilter/render.go (hunk 2 above)
Comment (shuofan) [reply modules/dop/component-protocol/components/requirement-task-overview/stackChartBlock/stackChartFilter/render.go:62]: not include requirement task overview

File: modules/dop/component-protocol/components/requirement-task-overview/container/simpleChart/render.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (Effet) [reply modules/dop/component-protocol/components/requirement-task-overview/container/simpleChart/render.go:49]: okay

File: modules/dop/component-protocol/components/requirement-task-overview/stackChartBlock/stackChartFilter/render.go (hunk 2 above)
Comment (Effet) [reply modules/dop/component-protocol/components/requirement-task-overview/stackChartBlock/stackChartFilter/render.go:62]: okay

**Review by Effet — APPROVED** [review]

**Pull request merged**